templates/admin/round_system_results.html.twig

```diff
@@ -0,0 +1,63 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+{% block content %}
+    <div class="container">
+        <h4>{{ tournament.title }}</h4>
+        <p>0 - чорні виграли, 1 - нічия, 2 - білі виграли</p>
+        <table class="table table-striped table-condensed">
+            <thead>
+            <tr>
+                <td> №</td>
+                <td>{{ 'Last name'|trans }}</td>
+                {% for participant in participants %}
+                    <td>{{ participant.participantOrder }}</td>
+                {% endfor %}
+            </tr>
+            </thead>
+            <tbody>
+
+            {% for participant_one in participants %}
+                <tr>
+                    <td>{{ participant_one.participantOrder }}</td>
+                    <td>{{ participant_one.player.lastName }}</td>
+
+                    {% for participant_two in participants %}
+                        <td>
+                            {% if participant_one == participant_two %}
+                                -
+                            {% else %}
+
+                                <a class="roundResultShowButton">
```

Comment:
It's preferable to name html classes like the following: round-result-show-button

Reply:
done